docs/skopeo-copy.1.md

```diff
@@ -58,6 +58,10 @@ If the authorization state is not found there, $HOME/.docker/config.json is chec
 
 Existing signatures, if any, are preserved as well.
 
+**--dest-compress-format** _format_ Specifies the compression format to use.
```

Comment:
Still needs to document the available values, or maybe how to discover them.

Reply:
I'll list the supported algorithms